src/selic-entries.ts

```diff
@@ -0,0 +1,11 @@
+export async function selicEntries(initialDate: string, finalDate: string) {
+    const url = new URL('https://api.bcb.gov.br/dados/serie/bcdata.sgs.11/dados?formato=json');
+    const params = new URLSearchParams;
```

Comment:
You missed parenthesis when instantiating a new instance of a class. I'm surprised that that worked.

```suggestion
    const params = new URLSearchParams();
```